providers/standard/tests/provider_tests/standard/utils/test_python_virtualenv.py

```diff
@@ -192,25 +193,77 @@ def test_should_create_virtualenv_with_extra_packages_uv(self, mock_execute_in_s
         )
 
     def test_remove_task_decorator(self):
```

Comment:
nits: we probably could refactor these cases through `pytest.mark.parametrize` (remember to use id to make it more readable)